flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/cli/SqlMultiLineParser.java

```diff
@@ -31,18 +37,61 @@
  */
 public class SqlMultiLineParser extends DefaultParser {
```

Comment:
This can be package private

Reply:
I think we should by default try to hide as much as possible. In this case, this class is not used outside this package (and doesn't really have any specific utility outside sql client anyway), so we should make it package private

Reply:
if you really think it is necessary, I can remove the `public`